evalai/utils/urls.py

```diff
@@ -1,12 +1,12 @@
 from enum import Enum
 
-
+GET_ALL_CHALLENGES_URL = "/api/challenges/challenge/time/{}/approved/{}/published/{}"
```

Comment:
Please move the URLs here - https://github.com/Cloud-CV/evalai-cli/blob/master/evalai/utils/urls.py#L4